.nvmrc

```diff
@@ -0,0 +1 @@
+8.11.0
```

Comment:
Why do we need that?
https://github.com/neos/neos-ui/blob/master/package.json#L37